source/client-side-encryption/etc/data/range-encryptedFields-Date.json

```
@@ -0,0 +1,30 @@
+{
+    "fields": [
+      {
+        "keyId": {
+          "$binary": {
+            "base64": "EjRWeBI0mHYSNBI0VniQEg==",
+            "subType": "04"
+          }
+        },
+        "path": "encryptedDate",
+        "bsonType": "date",
+        "queries": {
+          "queryType": "rangePreview",
+          "sparsity": {
+            "$numberInt": "1"
```

```suggestion
            "$numberLong": "1"
```

Please update `sparsity` to numberLong in the .json files. This was also fixed in https://github.com/mongodb/specifications/pull/1352/commits/b5ffac11aa7ddf189141774b1e53b6691f833730. The Go driver specification test runner accepts either type, C# does not.